.github/workflows/csi_operator_ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: Community Operators test
+on:
+  push:
+    branches:
+      - task/CSI-2892_cert_community_operator_hub
+jobs:
+  operator_image_build:
+   name: "set env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+   outputs:
+     operator_image: "csiblock1/ibm-block-csi-operator-amd64:new"
```

Comment:
for now I think we should use latest GA image, instead of this one.
could you open a ticket to build and push images in GitHub and write its number here?
(so this value would be dynamic in the future)